zeus/monitor/rapl.py

```diff
@@ -0,0 +1,146 @@
+"""Monitor the wrapping around of RAPL counters."""
+
+from __future__ import annotations
+
+import atexit
+import typing
+import tempfile
+import os
+from time import time, sleep
+import multiprocessing as mp
+
+import pandas as pd
+from sklearn.metrics import auc
+
+from zeus.utils.logging import get_logger
+from zeus.device import get_gpus
+
+def infer_counter_update_period(rapl_file_path: str) -> float: 
+    """Determine how long to sleep until next the next poll
+
+    """
+    return 1.0
+
+class RaplMonitor:
```

Comment:
Is `RaplMonitor` the best name? Maybe something like `RaplWraparoundTracker` is more specific. Calling it a RAPL monitor makes it sound like it actually tracks the energy counter values.